lint-python/action.yml

```diff
@@ -1,20 +1,41 @@
 name: Lint Python
 description: Linting for Python projects
+
 inputs:
   exclude_trailing_comma_path:
     required: false
     description: Ignore trailing comma changes in given path
     default: ''
+  src:
+    required: false
+    description: Source to run ruff
+    default: '.'
 
 runs:
   using: composite
   steps:
-    - run: find ${{ inputs.exclude_trailing_comma_path }} -name '*.py' | xargs poetry run add-trailing-comma --exit-zero-even-if-changed --py36-plus
-      if: ${{ inputs.exclude_trailing_comma_path != '' }}
-      shell: bash
+      - run: echo "ruff-available=$(grep ruff pyproject.toml | wc -l)" >> $GITHUB_OUTPUT
```

Comment:
We can keep it like this if it's a temporary solution. But there is another way to check if package is available: `poetry show ruff`. The command returns non-zero status code if the package is not available.

Reply:
Good idea, changed it to `poetry show ruff -q ; echo $?`